Sources/xcproj/PBXBuildFile.swift

```diff
@@ -88,11 +88,9 @@ extension PBXBuildFile: PlistSerializable {
     }
 
     private func name(fileRef: String, proj: PBXProj) -> String? {
-        let fileReference = proj.fileReferences.getReference(fileRef)
```

Comment:
I didn't know we still had some objects with methods to get the names for the comments. Would you mind using the methods from `PBXProj+Helpers.swift`? I can also do it on another PR.

Reply:
Yeah, I noticed that there are also functions in PBXProj `getName` and `getConfig`. `getName` actually mirrors the one in Helpers but has a slightly different implementation. I've removed these now